personal_assistant/pa_tag/admin.py

```diff
@@ -0,0 +1,4 @@
+from django.contrib import admin
+from .models import Tag
```

Comment:
Роби пропуск між власними і сторонніми залежностями:
```suggestion
from django.contrib import admin

from .models import Tag
```